CONTRIBUTING.rst

```diff
@@ -64,24 +64,13 @@ Using Make Tasks for Development Environment Setup
     $ git push origin name-of-your-bugfix-or-feature
 
 8. Submit a pull request through the GitHub website.
+9. Clean-up your environment::
 
-Doing it the Hard Way
```

Comment:
There is still a reference to this section on line 15 of the file.

Reply:
Updated to remove reference.